### PR TITLE
Move simulatePreimages to utils.test

### PR DESCRIPTION
**Pull request by Geod24**

### CODE REVIEW

**Comment by Geod24** [comment]
Now with added süß.

File: source/agora/utils/Test.d [hunk 1]
```diff
@@ -442,3 +444,34 @@ unittest
...
+                     This can be used to test slashing.
+
+*******************************************************************************/
+
+public void simulatePreimages (Ledger ledger, in Height height,
+                               uint[] skip_indexes = null) @safe
+{
+    ledger.getValidators(height).enumerate.each!((idx, val)
+    {
+        if (skip_indexes.length == 0 || !skip_indexes.canFind(idx))
```
Comment (linked0) [inline source/agora/utils/Test.d]:
```suggestion
        if (!skip_indexes.canFind(idx))
```
I think there is some reason to add the condition more though.

**Review by linked0 — APPROVED** [review]
LGTM

File: source/agora/utils/Test.d (hunk 1 above)
Comment (Geod24) [reply source/agora/utils/Test.d]: Added as a separate commit to avoid moving and modifying.

**Pull request merged**